nf-example-topology/package-context.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kptfile.kpt.dev
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  name: example
```

Comment:
Just as a clarification, leaving name "example" here is fine - this value is overwritten when we `kpt pkg get --for-deployment` or clone to a Porch deployment repo.

Reply:
OK, thanks for explaining.